Made a button to make/view reviews in game detail page

danielngira opened the pull request:
Related Issue: #261 

Tasks I completed in this PR:

- Made a form ReviewForm for creating new reviews
- Updated the GameDetailView with a FormMixin and LoginRequiredMixin to allow logged-in users to create reviews for games within the game detail page.
- Made a ReviewListView and a game-review template. Also updated the URLs to include a URL to the game-reviews page for a specific game
- Updated the game-detail template page to include a form to create new reviews

This PR depends on PR #274 that will create a new review entity in the models

danielngira committed:
Updates to forms.py by black code formatter

danielngira committed:
black formatter urls.py

danielngira committed:
Improvements to views

danielngira committed:
Linter changes to game_detail.html

danielngira committed:
Improvements to game_reviews.html

danielngira committed:
issues

danielngira committed:
Fixed style issues

danielngira commented:
https://github.com/uchicago-cs/chigame/assets/81624602/eadcd678-12ef-4159-ac42-0d7bb969694f

You can now leave a review on a game and view a list of a game's public reviews.

rakrusch commented on src/chigame/games/forms.py:
```diff
@@ -25,3 +26,26 @@ class Meta:
         widgets = {"name": forms.TextInput}
+
+
+class ReviewForm(forms.ModelForm):
```
Review form here looks good to me!

rakrusch commented on src/chigame/games/views.py:
```diff
@@ -42,10 +43,34 @@ def get_queryset(self):
 
 
-class GameDetailView(DetailView):
+class GameDetailView(LoginRequiredMixin, FormMixin, DetailView):
```
Nice job adding reviews onto game detail view!

danielngira committed:
Fixed style issues

rakrusch reviewed (approved):
Everything looks good and appears to be working well. Nice job Daniel!

danielngira commented:
I noticed the template page was not displaying the game's name. In my latest commit, I fixed how it is rendered in the `ReviewListView`

majorsylvie committed:
Merge branch 'dev' into game-management/reviews-page

majorsylvie reviewed (changes requested):
If not supplying anything to a review, the validation error shows twice: 
<img width="377" alt="image" src="https://github.com/uchicago-cs/chigame/assets/86740254/a953ee24-0348-446a-aa72-39d26eceb3c8">

The titles of reviews are not seen in the list view:
<img width="494" alt="image" src="https://github.com/uchicago-cs/chigame/assets/86740254/5b4bb2b3-2c02-47c1-acd2-88b81a582d07">

Seen more obviously with: 
<img width="460" alt="image" src="https://github.com/uchicago-cs/chigame/assets/86740254/ec541708-e8a8-44b8-925b-3967102320a3">

<img width="543" alt="image" src="https://github.com/uchicago-cs/chigame/assets/86740254/f67d2868-914b-4822-a26a-eadafc1d0689">

Along with that, the numbers are displayed somewhat oddly. 
You use the `/5` instead of `/5.00` even though we are doing 2 decimal place floats from 1-5 for ratings.

Along with that, you still display the rating for reviews without one, leading to this odd feature of the UI:
<img width="425" alt="image" src="https://github.com/uchicago-cs/chigame/assets/86740254/7dd42af9-7786-4d6f-8e94-71ce02db5ebe">

**Please address the following:**
- not displaying `None` ratings
- changing the ratings to show `/5.00` to better reflect the numbers for ratings
- Actually display rating titles if they exist in the ratings-per-game list view

**Issues to create based on enhancing the work in this PR**
- one to add a visual display (such as filled in stars) to reflect the rating 
  - ie: 4.5 == 4 and a half filled in stars
- remove duplicate error messages for review validation
- add extra styling/CSS to the reviews in the review-per-game list view to make them look nicer (whatever that means to you).

Please make the requested changes **and create issues in the backlog for all listed enhancements**

Thank you for your work! this is super cool functionality that you've spearheaded @danielngira. This is quite close to merging!

majorsylvie committed:
Merge branch 'dev' into game-management/reviews-page

danielngira committed:
Updated game reviews list template page

danielngira committed:
Merge branch 'game-management/reviews-page' of https://github.com/uchicago-cs/chigame into game-management/reviews-page

danielngira committed:
Made migrations

danielngira committed:
Merge branch 'dev' into game-management/reviews-page

danielngira commented:
> If not supplying anything to a review, the validation error shows twice: <img alt="image" width="377" src="https://private-user-images.githubusercontent.com/86740254/287890324-a953ee24-0348-446a-aa72-39d26eceb3c8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE3NDE5NjQsIm5iZiI6MTcwMTc0MTY2NCwicGF0aCI6Ii84Njc0MDI1NC8yODc4OTAzMjQtYTk1M2VlMjQtMDM0OC00NDZhLWFhNzItMzlkMjZlY2ViM2M4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjA1VDAyMDEwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ1ZjljNDUxYmQ3YmMwYjc3YmJiZTBmZDkzNDMwNjYzYWYxODgwOWMxZmNjMzNmNjk3YmZlOGFhYzIzZTQ1N2YmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.-nhAuwi5T58e5wM69Z5qIfNa6nKauxIR-D7_Z6LiFw0">
> 
> The titles of reviews are not seen in the list view: <img alt="image" width="494" src="https://private-user-images.githubusercontent.com/86740254/287890526-5b4bb2b3-2c02-47c1-acd2-88b81a582d07.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE3NDE5NjQsIm5iZiI6MTcwMTc0MTY2NCwicGF0aCI6Ii84Njc0MDI1NC8yODc4OTA1MjYtNWI0YmIyYjMtMmMwMi00N2MxLWFjZDItODhiODFhNTgyZDA3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjA1VDAyMDEwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQyZjUzOTNhMGFlYmY3OWJjNjVlY2E5OWIyMWRlY2FhNGVmMjM3N2RmYzQ1Mzk4ZDJiMzU5ZGMwNzEwZWQxOTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.BGaeDeZzgs3RfVYplDEMQQ5vd2en7whHri7fM5GjW98">
> 
> Seen more obviously with: <img alt="image" width="460" src="https://private-user-images.githubusercontent.com/86740254/287890737-ec541708-e8a8-44b8-925b-3967102320a3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE3NDE5NjQsIm5iZiI6MTcwMTc0MTY2NCwicGF0aCI6Ii84Njc0MDI1NC8yODc4OTA3MzctZWM1NDE3MDgtZThhOC00NGI4LTkyNWItMzk2NzEwMjMyMGEzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjA1VDAyMDEwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEyZDBkYjA3YjM2YjViZTVlNWZhYzM2NmNkNjBiODA5ZTQ0ZTVhY2U5NjVlNTEwNzBmMzRmMzY3ZWNhYzQyYTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Xv2QNoBTkOsqUoYQlo8iX19b-7C41JRPZtN2gCvcVFI">
> 
> <img alt="image" width="543" src="https://private-user-images.githubusercontent.com/86740254/287890726-f67d2868-914b-4822-a26a-eadafc1d0689.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE3NDE5NjQsIm5iZiI6MTcwMTc0MTY2NCwicGF0aCI6Ii84Njc0MDI1NC8yODc4OTA3MjYtZjY3ZDI4NjgtOTE0Yi00ODIyLWEyNmEtZWFkYWZjMWQwNjg5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjA1VDAyMDEwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVmNmUxNzkwMjA0MjUxM2ZhN2U0MzkyZjIwZGY1ZjM2MDdjZGYxOWVkYTVmNzUyNzFlNjBkZWYwMGMzOTgyMWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.6PR3A8X_iaTXb1mSLKYe7UIa8nIbVVo2iMpElZC124Y">
> Along with that, the numbers are displayed somewhat oddly. You use the `/5` instead of `/5.00` even though we are doing 2 decimal place floats from 1-5 for ratings.
> 
> Along with that, you still display the rating for reviews without one, leading to this odd feature of the UI: <img alt="image" width="425" src="https://private-user-images.githubusercontent.com/86740254/287890901-7dd42af9-7786-4d6f-8e94-71ce02db5ebe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE3NDE5NjQsIm5iZiI6MTcwMTc0MTY2NCwicGF0aCI6Ii84Njc0MDI1NC8yODc4OTA5MDEtN2RkNDJhZjktNzc4Ni00ZDZmLThlOTQtNzFjZTAyZGI1ZWJlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjA1VDAyMDEwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA3YzIyOTBiNzg1ODUxZjFhMGE1MzM0ZTE5NDNhNTlhOGNkZjY3MjcwOWRkYjU4MmU3NTkyZDUwNWNiZDgxNGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.-Fc5Zkfre8t-esOZRh8OkXZftn0jOTgX5hiUvjTjo8o">
> 
> **Please address the following:**
> 
> * not displaying `None` ratings
> * changing the ratings to show `/5.00` to better reflect the numbers for ratings
> * Actually display rating titles if they exist in the ratings-per-game list view
> 
> **Issues to create based on enhancing the work in this PR**
> 
> * one to add a visual display (such as filled in stars) to reflect the rating
>   
>   * ie: 4.5 == 4 and a half filled in stars
> * remove duplicate error messages for review validation
> * add extra styling/CSS to the reviews in the review-per-game list view to make them look nicer (whatever that means to you).
> 
> Please make the requested changes **and create issues in the backlog for all listed enhancements**
> 
> Thank you for your work! this is super cool functionality that you've spearheaded @danielngira. This is quite close to merging!



> If not supplying anything to a review, the validation error shows twice: <img alt="image" width="377" src="https://private-user-images.githubusercontent.com/86740254/287890324-a953ee24-0348-446a-aa72-39d26eceb3c8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE3NDI5NjksIm5iZiI6MTcwMTc0MjY2OSwicGF0aCI6Ii84Njc0MDI1NC8yODc4OTAzMjQtYTk1M2VlMjQtMDM0OC00NDZhLWFhNzItMzlkMjZlY2ViM2M4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjA1VDAyMTc0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcyMDI5Y2NiZjdiM2Y3YzdkMTRiOGIwMTRmYTczMmIwMGUxZmI1YTg4M2YwMzYyMGVhNzEyNjI2ZjViOGYyOWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.FidV_8yfbKBDqNp7UJuLhmSd4uFpmYF4bTdfWmw6HF0">
> 
> The titles of reviews are not seen in the list view: <img alt="image" width="494" src="https://private-user-images.githubusercontent.com/86740254/287890526-5b4bb2b3-2c02-47c1-acd2-88b81a582d07.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE3NDI5NjksIm5iZiI6MTcwMTc0MjY2OSwicGF0aCI6Ii84Njc0MDI1NC8yODc4OTA1MjYtNWI0YmIyYjMtMmMwMi00N2MxLWFjZDItODhiODFhNTgyZDA3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjA1VDAyMTc0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU3NzA3YjMxYWZhOWNhMjQ4YTRiNTM4MjMyMzI0OWJlZjc1MjIwOWQyZDAwZTQ2NTMzMmZhZTA4YWI2MmVjNjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.4IR7uyTM_QKosCdsGoARbEGZmbJ1Kshp4zhlnB80QVE">
> 
> Seen more obviously with: <img alt="image" width="460" src="https://private-user-images.githubusercontent.com/86740254/287890737-ec541708-e8a8-44b8-925b-3967102320a3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE3NDI5NjksIm5iZiI6MTcwMTc0MjY2OSwicGF0aCI6Ii84Njc0MDI1NC8yODc4OTA3MzctZWM1NDE3MDgtZThhOC00NGI4LTkyNWItMzk2NzEwMjMyMGEzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjA1VDAyMTc0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU3ZGNmODFmMzVmMjViYjMzZTRiYjlhMTM1NzFkNzhhMzI5YzM4ZjVmZWM3Zjc4YjMwZjJlZWYxNDAyMWFkMDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.8kh1b1b8OpxeJkQ57kbehYYAgcvKQCcQgaAsi8NeGSU">
> 
> <img alt="image" width="543" src="https://private-user-images.githubusercontent.com/86740254/287890726-f67d2868-914b-4822-a26a-eadafc1d0689.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE3NDI5NjksIm5iZiI6MTcwMTc0MjY2OSwicGF0aCI6Ii84Njc0MDI1NC8yODc4OTA3MjYtZjY3ZDI4NjgtOTE0Yi00ODIyLWEyNmEtZWFkYWZjMWQwNjg5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjA1VDAyMTc0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk1NWQwMDg2MWQ4NGE2ODNlMTcyODYzZTIxZmM1NjRmN2Y3ODZlN2UwMGNjZmVmMTdjOGVhMDk2Mjc1ZDZhZTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.TJF6rZI7X4mivUOMtz67sOh_WuIRLQAZMjICSnWEs74">
> Along with that, the numbers are displayed somewhat oddly. You use the `/5` instead of `/5.00` even though we are doing 2 decimal place floats from 1-5 for ratings.
> 
> Along with that, you still display the rating for reviews without one, leading to this odd feature of the UI: <img alt="image" width="425" src="https://private-user-images.githubusercontent.com/86740254/287890901-7dd42af9-7786-4d6f-8e94-71ce02db5ebe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE3NDI5NjksIm5iZiI6MTcwMTc0MjY2OSwicGF0aCI6Ii84Njc0MDI1NC8yODc4OTA5MDEtN2RkNDJhZjktNzc4Ni00ZDZmLThlOTQtNzFjZTAyZGI1ZWJlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjA1VDAyMTc0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZlNDU5NjJhM2IxZTc2OWE1ZWI5NDAyNTA2ODM2MTBiYzNkZDVmY2RjMTAyNjVjNjljYzg2YTRlM2ZjZTg1ZjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.eXu6-t6WRfls1lDB88-z-97q0pXVTGBF4JwHDFksdRc">
> 
> **Please address the following:**
> 
> * not displaying `None` ratings
> * changing the ratings to show `/5.00` to better reflect the numbers for ratings
> * Actually display rating titles if they exist in the ratings-per-game list view
> 
> **Issues to create based on enhancing the work in this PR**
> 
> * one to add a visual display (such as filled in stars) to reflect the rating
>   
>   * ie: 4.5 == 4 and a half filled in stars
> * remove duplicate error messages for review validation
> * add extra styling/CSS to the reviews in the review-per-game list view to make them look nicer (whatever that means to you).
> 
> Please make the requested changes **and create issues in the backlog for all listed enhancements**
> 
> Thank you for your work! this is super cool functionality that you've spearheaded @danielngira. This is quite close to merging!

Thank you! I have fixed the issue where the reviews list page showed NULL ratings. I also added the reviews title to the the page. I also changed the ratings to be placed </5.00>. Will work on making the backlog issues.

danielngira committed:
Merge branch 'dev' into game-management/reviews-page

danielngira committed:
fixed merge conflicts

danielngira commented:
In my latest commit, I fixed a merge conflict in the `views.py` file

majorsylvie committed:
Merge branch 'dev' into game-management/reviews-page

majorsylvie reviewed (approved):
You've successfully added the title, `5.00`, and removed the rating text for non-existent ratings! 

Great work! Lets get this merged.

majorsylvie commented:
Issue Score: Excellent

Comments:
Excellent work @danielngira ! 
You've got a descriptive summary and closing statement, you got and acted on feedback in your PR very well, you provided video evidence showing off your work.

Incredible job! 

Keep moving forward :)

<!-- {"score": "excellent", "minor": false} -->
